code/modules/projectiles/ammunition/magazines.dm

```diff
@@ -490,3 +490,12 @@
 /obj/item/ammo_box/magazine/laser/update_icon()
 	..()
 	icon_state = "[initial(icon_state)]-[Ceiling(ammo_count(0)/20)*20]"
+
+/obj/item/ammo_box/magazine/toy/smgm45
```

Comment:
Do normal donksofts have mags now or am I just being stupid

Reply:
They do have mags, and they are looking like the normal ones.